lib/common.py

```diff
@@ -139,6 +140,9 @@ def ssh(hostname_or_ip, cmd, check=True, simple_output=True, suppress_fingerprin
                 raise SSHCommandFailed(255, "SSH Error: %s" % banner_res.stdout.decode(), command)
             stdout = stdout[len(banner_res.stdout.decode()):]
 
+        if res.returncode:
+            logger.error("Got error {} '{}' while running '{}'".format(res.returncode, stdout, command))
+
         if check and res.returncode:
             raise SSHCommandFailed(res.returncode, stdout, command)
```

Comment:
I can do this if you prefer?
```py
        if res.returncode:
            logger.error("Got error {} '{}' while running '{}'".format(res.returncode, stdout, command))
            if check:
                raise SSHCommandFailed(res.returncode, stdout, command)
```

Reply:
```py
        if res.returncode:
            if check:
                logger.error("Got error {} '{}' while running '{}'".format(res.returncode, stdout, command))
                raise SSHCommandFailed(res.returncode, stdout, command)
            else:
                logger.warning("Got error {} '{}' while running '{}'".format(res.returncode, stdout, command))
```

Something like this?

Reply:
Maybe. :thinking: @stormi ?

Reply:
Sometimes we run commands that are expected to fail so that would not be a warning, rather a debug.